src/chapters/orm-relationships/many-to-one.rst

```diff
@@ -0,0 +1,205 @@
+
+.. sourcecode:: html
+   :lineno-start: 27
+
+   <div class="form-group">
+      <label>Category
+      <select th:field="${event.eventCategory}">
+         <option th:each="eventCategory : ${categories}"
+                  th:value="${eventCategory.id}"
+                  th:text="${eventCategory.name}"
+         ></option>
+      </select>
+      <p class="error" th:errors="${event.eventCategory}"></p>
+      </label>
+   </div>
+
+This new template code includes several changes:
+
+#. The ``select`` is now bound to the field ``eventCategory``.
+#. The loop references ``categories`` and has an updated iterator variable name, ``eventCategory``.
+#. The value of the ``select`` is now ``eventCategory.id``. This allows model binding to occur upon form submission. Spring Boot will determine the category object to assign to the new event object by referencing the ``id``.
+#. The display text for each ``option`` is now ``eventCategory.name``
+#. The error message display now references the new field, ``event.eventCategory``.
+
+The remaining usages of ``EventType`` refer to:
+
+#. An ``Event`` constructor parameter.
+#. The getter/setter pair for ``type`` in ``Event``.
+#. An import statement in ``EventController``.
+#. The ``EventType`` class itself.
+
+Removing this unneeded code resolves all remaining compiler errors. 
+
+Finally, the ``events/create.html`` template needs to be updated as well, since it still contains a reference to the ``type`` field of ``Event``:
+
+.. sourcecode:: html
+   :lineno-start: 23
+
+   <td th:text="${event.type}"></td>
+
+This usage wasn't found by IntelliJ because templates do not receive compile-time type checking like classes do. The updated version looks like this:
+
+.. sourcecode:: html
+   :lineno-start: 23
+
+   <td th:text="${event.eventCategory.name}"></td>
+
+Testing and Database Updates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Before starting up the, let's look at our ``event`` table:
+
+.. figure:: figures/event-table-before-update.png
+   :alt: The event table before starting up the application
+   :width: 600px
+
+   The ``event`` table will be updated the next time the app starts
+
+Notice that the ``type`` column remains, even though we have deleted the ``type`` field from the ``Event`` class. The next time we start up the application, Hibernate will attempt to update the schema to match the structure of our updated model class. 
+
+.. index:: ! truncate
+
+.. admonition:: Tip
+
+   Notice that existing events will not have a category, which violates our new ``@NotNull`` validation rule. The easiest way to handle this is to delete all rows in ``event``.
+   
+   To delete all events, truncate the ``event`` table in MySQL Workbench. (To truncate a SQL table is simply to delete all its rows.) To do this, select the table in the *Schemas* pane, right-click, and select *Truncate Table...*
```

Comment:
I'd put truncate in bold to follow our patterns for index entries and first mention